preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Typo:</p><blockquote>pr-processing</blockquote><p> </p><p>Update</p><blockquote> Since "Word" and "word" will essentially be considered <strong>as</strong> two different element in world representation</blockquote><p>To</p><blockquote> Since "Word" and "word" will be considered two different elements in world representation</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>